definitions/EiffelMetaProperty/3.1.0.yml

```diff
@@ -0,0 +1,210 @@
+            _format: >-
+              [A valid JWA RFC 7518 alg parameter value](https://tools.ietf.org/html/rfc7518#section-3.1),
+              excluding "none"  
+            type: string
+            enum:
+              - HS256
+              - HS384
+              - HS512
+              - RS256
+              - RS384
+              - RS512
+              - ES256
+              - ES384
+              - ES512
+              - PS256
+              - PS384
+              - PS512
+          publicKey:
+            _description: >-
+              The producer of the event may include the relevant public key
+              for convenience, rather than relying a separate key distribution
+              mechanism. Note that this property, along with the rest of the
+              event, is encompassed by the integrity protection offered via
+              __meta.security.integrityProtection__.
+            type: string
+        required:
+          - signature
+          - alg
+        additionalProperties: false
+      sequenceProtection:
+        _description: >-
+          An optional object for enabling verification of intact event
+          sequences in a distributed environment, thereby protecting against
+          data loss, race conditions and replay attacks. It allows event
+          publishers to state the order in which they produce a certain set
+          of events. In other words, it cannot provide any global guarantees
+          as to event sequencing, but rather per-publisher guarantees. Every
+          object in the array represents a named sequence of which this event
+          forms a part. For every event including a given named sequence,
+          the publisher SHALL increment
+          __meta.security.sequenceProtection.position__ by 1. The first event
+          produced in a given named sequence SHALL numbered 1.
+        type: array
+        items:
+          type: object
+          properties:
+            sequenceName:
+              _description: >-
+                The name of the sequence. There MUST not be two identical
+                __meta.security.sequenceProtection.sequenceName__ values in
+                the same event.
+              type: string
+            position:
+              _description: The number of the event within the named sequence.
+              type: integer
+          additionalProperties: false
+          required:
+            - sequenceName
+            - position
+    additionalProperties: false
+    required:
+      - authorIdentity
+  schemaUri:
+    _description: >-
+      A URI pointing at a location from where the schema used when creating this
+      event can be retrieved. It can be used to parse event data for validation
+      and extraction purposes, for example. Note, that the schema on that URI
+      should be considered immutable.
+    _format: URI
+    type: string
+    additionalProperties: false
```

Comment:
Does it really make sense to set additionalProperties for a string property?

Reply:
Of course, makes sense for object type only. Removed.